print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
```

Comment:
using "value" and "next" in conjunction with "this" seems confusing from a readability standpoint. what is this.next referring to?